package.json

```diff
@@ -37,6 +37,8 @@
     "prepublishOnly": "yarn run build"
   },
   "dependencies": {
+    "iron-session": "~8.0.2",
+    "jose": "~5.6.3",
```

Comment:
Should we strictly pin these or do we want leave the patch version open?

Reply:
I think leaving patches open is good practise.